src/content/docs/integrations/kubernetes-integration/installation/install-kubernetes-integration-using-helm.mdx

```diff
@@ -32,674 +34,258 @@ To install the Kubernetes integration using Helm, you will need your New Relic a
 1. Find and copy your [New Relic license key](/docs/accounts/install-new-relic/account-setup/license-key).
```

Comment:
Should we point to the wizard as well?

Reply:
I think this is already done in the "Install kubernetes" parent page.